src/SFA.DAS.EmployerCommitmentsV2.Web/Controllers/CohortController.cs

```diff
@@ -254,6 +258,20 @@ public async Task<IActionResult> Apprentice(ApprenticeViewModel model)
             var request = await _modelMapper.Map<CreateCohortRequest>(model);
             var newCohort = await _commitmentsApiClient.CreateCohort(request);
 
+            var cohortApprenticeships = await _commitmentsApiClient.GetDraftApprenticeships(newCohort.CohortId);
+
+            if (cohortApprenticeships.DraftApprenticeships.Count() == 1)
```

Comment:
Can probably change this to `SingleOrDefault` which gets you 1 if there is only 1 , or null if not, would then simplify your code below.

```
if(draftApprenticeship?.CourseCode != null)
{
  // encode and redirect
}
```